tools/clang/test/CodeGenSPIRV/method.struct.method.hlsl

```diff
@@ -48,51 +59,63 @@ struct T {
 // CHECK-NEXT:           %s = OpVariable %_ptr_Function_S Function
 // CHECK-NEXT:           %t = OpVariable %_ptr_Function_T Function
 // CHECK-NEXT: %param_var_c = OpVariable %_ptr_Function_float Function
-// CHECK:          {{%\d+}} = OpFunctionCall %float %fn_no_ref %s
-// CHECK:          {{%\d+}} = OpFunctionCall %float %fn_ref %s
-// CHECK:          {{%\d+}} = OpFunctionCall %float %fn_call %s %param_var_c
-// CHECK:          {{%\d+}} = OpFunctionCall %float %fn_nested %t
+// CHECK:          {{%\d+}} = OpFunctionCall %float %S_fn_no_ref %s
+// CHECK:          {{%\d+}} = OpFunctionCall %float %S_fn_ref %s
+// CHECK:          {{%\d+}} = OpFunctionCall %float %S_fn_call %s %param_var_c
+// CHECK:          {{%\d+}} = OpFunctionCall %float %T_fn_nested %t
+// CHECK:          {{%\d+}} = OpFunctionCall %float %S_fn_static
+// CHECK:          {{%\d+}} = OpFunctionCall %float %T_fn_static
 // CHECK:                     OpFunctionEnd
 float main() : A {
     S s;
     T t;
-    return s.fn_no_ref() + s.fn_ref() + s.fn_call(5.0) + t.fn_nested();
+    return s.fn_no_ref() + s.fn_ref() + s.fn_call(5.0) + t.fn_nested() +
+           s.fn_static() + t.fn_static();
```

Comment:
Would you also add ``S::fn_static()`` or ``T::fn_static()`` to the test?

Reply:
Done